src/api/hooks/products.ts

```diff
@@ -0,0 +1,21 @@
+import { useQuery } from '@tanstack/react-query';
+
+import type { ProductOptionsData } from '@/types';
+
+import { fetchInstance } from '../instance';
+
+const getProductOptionsPath = (productId: string) => `/v1/products/${productId}/options`;
+
+export const getProductOptions = async (productId: string): Promise<ProductOptionsData> => {
+  const response = await fetchInstance.get<ProductOptionsData>(getProductOptionsPath(productId));
+  return response.data;
+};
+
+export const useGetProductOptions = (productId: string) => {
+  return useQuery<ProductOptionsData, Error>({
+    queryKey: ['productOptions', productId],
+    queryFn: () => getProductOptions(productId),
+    staleTime: 60000,
```

Comment:
numeric seprator를 사용해보시면 좋겠네요 ~
```suggestion
    staleTime: 60_000,
```

Reply:
그리고 staleTime에 60,000이 들어간 배경이 궁금하네요 ㅎㅎ

Reply:
staleTime을 설정한 이유는 데이터가 유지되는 시간을 설정하여, 해당 시간 동안 캐시된 데이터를 사용해서 불필요한 서버 요청을 줄이고, 사용자에게 빠른 응답을 제공할 수 있기 때문입니다.